generic/ras.py

```diff
@@ -40,6 +40,14 @@ def setUp(self):
         architecture = os.uname()[4]
         if "ppc" not in architecture:
             self.skip("supported only on Power platform")
+        if 'PowerNV' in process.system_output('grep ^platform /proc/cpuinfo'):
+            nv_flag = False
+            for cmd in ['lsmcode', 'lsprop', 'nvram', 'ppc64_cpu']:
+                if cmd in self.name.name:
```

Comment:
self.name.name variable I didn't see any where in 'ras.py' 
is this a special variable or a typo?